__tests__/data/versions-manifest.json

```diff
@@ -0,0 +1,152 @@
+[
```

Comment:
this is a minimal version of versions-manifest @ https://github.com/actions/node-versions/ - just enough versions to unit test resolution